app/operations/shopify/product_operations.rb

```diff
@@ -0,0 +1,27 @@
+module Shopify
+  class ProductOperations
+    class << self
+      def create(spree_product:)
+        service = Shopify::ProductExporter.new(spree_product: spree_product)
+        service.perform
+      end
+
+      def update(spree_product:)
```

Comment:
With these taking only one argument, is the keyword arg a requirement? In addition, we are not setting these keyword args to anything in the args thereby they are just regular args correct?